static/js/util/util_test.js

```diff
@@ -325,10 +325,17 @@ describe('utility functions', () => {
     });
 
     it('should return fields for dialog for an empty profile', () => {
-      let expectation = [false, {
-        url: "/profile/personal",
-        title: "Personal Info",
-        text: "Please complete your personal information.",
+      let expectation = [false, "/profile/personal", {
+        first_name: 'Given name is required',
+        last_name: 'Family name is required',
+        preferred_name: "Preferred name is required",
+        gender: "Gender is required",
+        preferred_language: "Preferred language is required",
+        city: "City is required",
+        country: "Country is required",
+        birth_city: "City is required",
+        birth_country: "Country is required",
+        date_of_birth: "Date of birth is required",
```

Comment:
actually it is different. please see the ``PersonalTab.defaultProps.validationMessages``.The **required** string is missing in ``validationMessages``.

```validationMessages: {
      'first_name': "Given name",
      'last_name': "Family name",
      'preferred_name': "Preferred name",
      'gender': "Gender",
      'preferred_language': "Preferred language",
      'city': "City",
      'state_or_territory': 'State or Territory',
      'country': "Country",
      'birth_city': 'City',
      'birth_state_or_territory': 'State or Territory',
      'birth_country': "Country",
      'date_of_birth': "Date of birth"
    }```